### DISCUSSION TITLE
wallpaper directory

### DISCUSSION THREAD

**Discussion by TMasser1** [prasanthrangan/hyprdots#1941] [Q&A]
what is the Hyde wallpaper directory i want to add my custom one but not form the ./hyde.conf i want to add the wallpaper to the theme menu but it seems that i cannot find the directory of these wallpaper that are in the SUPER+SHIFT+W menu is there any thing that i should do even i unhidden the hidden files i found the .themes folder but it does not have the wallpaper directory that im searching for

**Comment by kRHYME7** [comment]
Stored per theme in  `~/.config/hyde/themes/THEME_NAME/wallpapers`

![image](https://github.com/user-attachments/assets/d00aefef-f28e-4ba6-9f75-2bbe9b582e10)
